R/plumber-step.R

```diff
@@ -162,7 +162,13 @@ PlumberEndpoint <- R6Class(
     #' @param req a request object
     #' @return a logical. `TRUE` when endpoint can serve request.
     canServe = function(req){
-      req$REQUEST_METHOD %in% self$verbs && !is.na(stri_match_first_regex(req$PATH_INFO, private$regex$regex)[1,1])
+      req$REQUEST_METHOD %in% self$verbs && self$matchesRoute(req$PATH_INFO)
```

Comment:
I think this is a spillover from 405, anyway this will fail most test as `matchesRoutes` is not `matchesPath` below.

Reply:
Thank you! I was confused there for a min.. haha